### PR TITLE
[System.Reflection] CoreFX import fot Pointer type

### PR DESCRIPTION
**Pull request by MaximLipnin**
Part of #9660

### CODE REVIEW

File: mcs/class/corlib/corert/Type.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace System
+{
+    partial class Type
+    {
+        public bool IsRuntimeImplemented() => true;
```
Comment (akoeplinger) [inline mcs/class/corlib/corert/Type.cs]: this should be internal

Comment (MaximLipnin) [reply mcs/class/corlib/corert/Type.cs]: addressed

**Comment by akoeplinger** [comment]
@monojenkins commit apidiff and csproj

**Comment by MaximLipnin** [comment]
Thanks for review!

**Pull request merged**